src/stats.js

```diff
@@ -76,7 +96,12 @@ module.exports = (common) => {
     })
 
     it('.bw Promise', () => {
-      return ipfs.stats.bw().then((res) => {
+      if (!withGo) {
+        console.log('Not supported in js-ipfs yet')
+        return
+      }
+
+      ipfs.stats.bw().then((res) => {
```

Comment:
@hacdias you should add a return here. Otherwise the test will pass before the call happens.

Reply:
OOPS! Sorry. fixed!